app/customer.py

```diff
@@ -0,0 +1,73 @@
+import dataclasses
+
+from app.shop import Shop
+
+
+@dataclasses.dataclass
+class Customer:
+    name: str
+    product_cart: dict
+    home_location: list[int]
+    money: int
+    car: dict
+    money_spent: int | float = 0
+    current_location: list[int] = None
+
+    def choose_shop_or_stay_home(
+            self,
+            shops_list: list[Shop],

```

Comment:
```suggestion
            shops: list[Shop],

```
Why you need suffix?